eras/conway/impl/src/Cardano/Ledger/Conway/Governance.hs

```diff
@@ -183,23 +187,22 @@ data EnactState era = EnactState
   }
   deriving (Generic)
 
-ensConstitutionL :: Lens' (EnactState era) (SafeHash (EraCrypto era) ByteString)
+ensConstitutionL :: Lens' (EnactState era) (Constitution era)
 ensConstitutionL = lens ensConstitution (\x y -> x {ensConstitution = y})
 
 instance EraPParams era => ToJSON (EnactState era) where
   toJSON = object . toEnactStatePairs
   toEncoding = pairs . mconcat . toEnactStatePairs
 
 toEnactStatePairs :: (KeyValue a, EraPParams era) => EnactState era -> [a]
-toEnactStatePairs cg@(EnactState _ _ _ _ _ _) =
```

Comment:
Can you pleas bring this back.

This is important, because pattern match like this protects us against forgetting to adjust the implementation when new fields are added.

Reply:
Oh, got it! Noted for henceforth!